packages/es-components/src/components/containers/tooltip/Tooltip.js

```diff
@@ -151,6 +152,7 @@ function Tooltip(props) {
             onClick={toggleShow}
             aria-describedby={descriptionId}
             styleType={styleType}
+            aria-label={buttonLabel}
```

Comment:
To complete the `linkProps` suggestion.

```suggestion
            {...linkProps}
```